src/app/components/navbar/navbar.component.ts

```diff
@@ -79,4 +81,10 @@ export class NavbarComponent implements OnInit, OnDestroy {
   private removeLeadingSlash(s = '') {
     return s.replace(/^\/+/g, '');
   }
+
+  //@doritrieur in order to get the first part of the path and compare to the route content.
+  private getFirstPathToken(s = '') {
```

Comment:
can you please change `s` to a more meaningful name?

Reply:
updated s to "domainString"